### PR TITLE
fix addkcmdline for deployed kernel only? #3532, refine the document

### PR DESCRIPTION
**Pull request by immarvin**
for issue https://github.com/xcat2/xcat-core/issues/3532#issuecomment-317825267

refine the description of bootparam.addkcmdline and linuximage.addkcmdline

The Doc:
http://10.3.5.21/xcat-doc/html/guides/admin-guides/references/man5/linuximage.5.html
http://10.3.5.21/xcat-doc/html/guides/admin-guides/references/man5/bootparams.5.html

### CODE REVIEW

File: docs/source/guides/admin-guides/references/man5/linuximage.5.rst [hunk 1]
```diff
@@ -56,7 +56,7 @@ linuximage Attributes:
 
 \ **addkcmdline**\ 
  
- User specified arguments to be passed to the kernel.  The user arguments are appended to xCAT.s default kernel arguments. For the kernel options need to be persistent after installation, specify them with prefix "R::".  This attribute is ignored if linuximage.boottarget is set.
+ User specified kernel options during provision. For the kernel options only for the provisioned system, specify them with prefix "R::". The options should be delimited with spaces(" ")
```
Comment (robin2008) [inline docs/source/guides/admin-guides/references/man5/linuximage.5.rst]: The message is still a bit confusing for the first sentence. Here the options are used to specify both network booting and OS kernel options after installed.

File: perl-xCAT/xCAT/Schema.pm [hunk 2]
```diff
@@ -799,7 +799,7 @@ passed as argument rather than by table value',
...
             boottarget => 'The name of the boottarget definition.  When this attribute is set, xCAT will use the kernel, initrd and kernel params defined in the boottarget definition instead of the default.',
-            addkcmdline => 'User specified arguments to be passed to the kernel.  The user arguments are appended to xCAT.s default kernel arguments. For the kernel options need to be persistent after installation, specify them with prefix "R::".  This attribute is ignored if linuximage.boottarget is set.',
+            addkcmdline => 'User specified kernel options during provision. For the kernel options only for the provisioned system, specify them with prefix "R::". The options should be delimited with spaces(" ")',
             pkglist => 'The fully qualified name of the file that stores the distro  packages list that will be included in the image. Make sure that if the pkgs in the pkglist have dependency pkgs, the dependency pkgs should be found in one of the pkgdir',
```
Comment (robin2008) [inline perl-xCAT/xCAT/Schema.pm]: why do we remove this line "This attribute is ignored if linuximage.boottarget is set."  Is it not valid anymore?

**Comment by immarvin** [comment]
updated

**Comment by robin2008** [comment]
The check failure is caused by ubuntu xcat-dep, no related with this PR.

**Pull request merged**